docs/IoT_quick_start/Dashboard.md

```diff
@@ -0,0 +1,37 @@
+# Dashboard
+
+The **Dashboard** displays widgets with visualized device data for your domain. Widgets are set and configured in the **Data model** tab of the **Device Center**, along with their corresponding **Observations** to enable collection of the reported device data. The **Dashboard** widgets are displayed per domain and feature different kinds of charts depending on the type of data collected.
+
+!!! important
+    The **Dashboard** and widgets are domain-wide, so they only apply to devices belonging to your domain. You can add up to 10 devices to a single widget, and they are added chronologically based on their **Creation time**. So if there are, for example, 15 devices in your domain, the 10 oldest of them will only be presented on a widget.
```

Comment:
I would like to add information that if your first 10 devices were not active within chosen time box in dashboard view  widgets will be all empty. Actual behaviour might be understood as bug but in fact it is not.

Reply:
We should also address question which my occur from end users: why my devices in theirs datamodel tabs have widgets configured even if didn't configure it. Answer is that widgets are domain domain-wide and each device will show widget as configured (even if it will not show up there because there are 10 devices which have connected earlier)

Reply:
We should also add information what is the range of supported objects for which widgets could be set up. Answer to this Question will be delivered by M.Grabowski. Please ask him in case I forgot.

Reply:
ok, added notes for the first two suggestions, waiting for info about the supported objects

Reply:
added note about supported objects